python/reg_tests/commonUtils.py

```diff
@@ -367,7 +367,7 @@ def standardTest(CFDSolver, ap, solve):
     reg_par_write_norm(resDot, 1e-10, 1e-10)
 
     parPrint('dFuncs/dw * wDot')
-    reg_root_write_dict(funcsDot, 1e-10, 1e-10)
+    reg_root_write_dict(funcsDot, 1e-9, 1e-9)
```

Comment:
does this have any implications? Will the tests that we are currently moving to other testing formats also contain this change?

Reply:
this was a mistake. Should not have been changed